core/extraction/towns_fund.py

```diff
@@ -492,6 +496,9 @@ def extract_funding_data(df_input: pd.DataFrame, project_lookup: dict) -> pd.Dat
     :param project_lookup: Dict of project_name / project_id mappings for this ingest.
     :return: A new DataFrame containing the extracted funding data.
     """
+
+    fund_type = list(project_lookup.values())[0].split("-")[0]
```

Comment:
Although probably not a problem with a dict this small, a more efficient (better practice) way to do this is
`next(iter(project_lookup.values())).split("-")[0]`
as it doesn't then do the relatively expensive transaction of casting to a full list of which you don't actually need any value other than the first

Reply:
Good to know, and added. I can see how this eis more effecient